python/packages/sdk/serverless_sdk/__init__.py

```diff
@@ -46,5 +50,13 @@ def _create_trace_span(
     ) -> trace.TraceSpan:
         return trace.TraceSpan(name, input, output, start_time, tags)
 
+    def capture_error(self, error, **kwargs) -> CapturedEvent:
```

Comment:
> But I don't think we can write integration test yet because we'd need the lambda sdk to actually make the call to capture_error.

True, integration test should be in follow up PR that concerns AWS Lambda SDK, and not part of SDK update 👍